Hi @AvtechScientific team,

Sure, here is `cmake` log:

```
-- The C compiler identification is GNU 5.3.0
-- The CXX compiler identification is GNU 5.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for CL_VERSION_2_0
-- Looking for CL_VERSION_2_0 - found
-- Found OpenCL: /usr/lib64/libOpenCL.so (found suitable version "2.0", minimum required is "1.1")
-- Boost version: 1.60.0
-- Found the following Boost libraries:
--   program_options
--   filesystem
--   system
CMake Warning at CMakeLists.txt:55 (find_package):
  Could not find a configuration file for package "VTK" that is compatible
  with requested version "6.1".

  The following configuration files were considered but not accepted:

    /usr/lib64/cmake/vtk-7.0/VTKConfig.cmake, version: 7.0.0
    /usr/lib/cmake/vtk-7.0/VTKConfig.cmake, version: 7.0.0
    /lib64/cmake/vtk-7.0/VTKConfig.cmake, version: 7.0.0
    /lib/cmake/vtk-7.0/VTKConfig.cmake, version: 7.0.0



CMake Error at CMakeLists.txt:56 (include):
  include called with wrong number of arguments.  include() only takes one
  file.


-- Found MATIO: /usr/lib64/libmatio.so (found suitable version "1.5.6", minimum required is "1.5.2")
-- Configuring incomplete, errors occurred!
See also "/home/arcanis/libasl/src/build/CMakeFiles/CMakeOutput.log".
```

[Here is](https://gist.github.com/arcan1s/931eefcaa688aa8ddf61) full CMakeOutput.log if you need. 

Sorry, I didn't take care about old version support, I've updated the pr to check vtk twice. I've run test and no error occurs:

```
Running tests...
Test project /home/arcanis/libasl/src/build
      Start  1: testABDFormat
 1/10 Test  #1: testABDFormat ....................   Passed    0.64 sec
      Start  2: testVectorOfElements
 2/10 Test  #2: testVectorOfElements .............   Passed    0.76 sec
      Start  3: testMatrixOfElements
 3/10 Test  #3: testMatrixOfElements .............   Passed    0.66 sec
      Start  4: testKernel
 4/10 Test  #4: testKernel .......................   Passed    1.17 sec
      Start  5: testOperators
 5/10 Test  #5: testOperators ....................   Passed    0.48 sec
      Start  6: testKernelMerger
 6/10 Test  #6: testKernelMerger .................   Passed    0.35 sec
      Start  7: testPrivateVar
 7/10 Test  #7: testPrivateVar ...................   Passed   88.90 sec
      Start  8: testASLData
 8/10 Test  #8: testASLData ......................   Passed    0.66 sec
      Start  9: testDistanceFunction
 9/10 Test  #9: testDistanceFunction .............   Passed    3.16 sec
      Start 10: testReductionFunction
10/10 Test #10: testReductionFunction ............   Passed    1.49 sec

100% tests passed, 0 tests failed out of 10

Label Time Summary:
DoublePrecision    =   5.83 sec (3 tests)
Performance        =  88.90 sec (1 test)

Total Test time (real) =  98.27 sec
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,7 +54,7 @@ find_package(OpenCL 1.1 REQUIRED)
 find_package(Boost 1.55 REQUIRED COMPONENTS program_options filesystem system)
 find_package(VTK 6.1 QUIET COMPONENTS vtkRenderingCore vtkImagingCore vtkFiltersCore vtkIOCore vtkIOLegacy vtkIOXML vtkIOMINC vtkCommonCore vtkViewsCore vtkftgl vtksys vtkDICOMParser vtkexpat vtkzlib NO_MODULE)
 if(NOT VTK_FOUND)
-	find_package(VTK 7.0 QUIET COMPONENTS vtkRenderingCore vtkImagingCore vtkFiltersCore vtkIOCore vtkIOLegacy vtkIOXML vtkIOMINC vtkCommonCore vtkViewsCore vtksys vtkDICOMParser vtkexpat vtkzlib NO_MODULE)
+	find_package(VTK 7.0 REQUIRED COMPONENTS vtkRenderingCore vtkImagingCore vtkFiltersCore vtkIOCore vtkIOLegacy vtkIOXML vtkIOMINC vtkCommonCore vtkViewsCore vtksys vtkDICOMParser vtkexpat vtkzlib NO_MODULE)
 endif()
 include(${VTK_USE_FILE})
 

